### ISSUE TITLE
ENH: changes after first revision

### ISSUE THREAD

**Issue by Javicadserres** [Javicadserres/wind-production-forecast#2]
- [x] Change interval period of the data. New interval: 2015-2018.
- [x] Add to the input data the day of the year, month and hour. 
- [x] Add new function cost funtion: Smooth pinball loss (see [here](https://arxiv.org/pdf/1710.01720.pdf)).
- [x] Add scores: QS, ACE, IS, PICP and wide.
- [x] Add benchmark: Linear model x8. One model for each quantile [0.025, 0.05, 0.1, 0.15]
- [x] Spread vs Non Spread
- [ ] Hyperparameters: Layers [3, 4, 5], number of hidden neurons [50, 100, 150], learning rate [1e-5, 1e-6, 1e-7]
